packages/user-management/spec/it/standard-user-workflow/core/account/StandardAccountConfirmation.spec.ts

```diff
@@ -0,0 +1,156 @@
+        this.successUrl = this.mockServer.urlFor('/home');
+        this.failureUrl = this.mockServer.urlFor('/confirmation-failed');
+        await given()
+          .credentials()
+          /**/ .fromEnv()
+          /**/ .newApp()
+          /**/ .and()
+          .worker()
+          /**/ .testModule(async () => {
+            mockAxios.onPost('mailjet-url').reply(200, {});
+            when(tokenGenerator.generate()).thenResolve({ value: '123456' });
+            when(uuidGenerator.generate()).thenResolve({ value: '42' });
+
+            // create configurer
+            const configurer = new StandardUserWorkflowConfigurerImpl(properties, zetapushContext);
+            configurer
+              .registration()
+              /**/ .account()
+              /*  */ .uuid()
+              /*    */ .generator(instance(uuidGenerator))
+              /*    */ .and()
+              /*  */ .initialStatus()
+              /*    */ .value(StandardAccountStatus.WaitingConfirmation)
+              /*    */ .and()
+              /*  */ .storage(LegacyAdapterUserRepository)
+              /*  */ .and()
+              // /*  */ .fields()
+              // /*    */.scan()
+              // /*      */.annotations()
+              // /*      */.and()
+              // /*    */.and()
+              /**/ .confirmation()
+              /*  */ .url(confirmationUrl)
+              /*  */ .token()
+              /*    */ .generator(instance(tokenGenerator))
+              /*    */ .validity(5000)
+              /*    */ .storage(GdaTokenRepository)
+              /*    */ .and()
+              /*  */ .email()
+              /*    */ .mailjet()
+              /*      */ .apiKeyPublic('public-key')
+              /*      */ .apiKeyPrivate('private-key')
+              /*      */ .url('mailjet-url')
+              /*      */ .httpClient(axiosInstance)
+              /*      */ .and()
+              /*    */ .from('no-reply@zetapush.com')
+              /*    */ .subject(`Confirm your inscription`)
+              /*    */ .htmlTemplate()
+              /*      */ .template(htmlTemplate)
+              /*      */ .and()
+              /*    */ .textTemplate()
+              /*      */ .template(textTemplate)
+              /*      */ .and()
+              /*    */ .and()
+              /*  */ .redirection()
+              /*    */ .successUrl(this.successUrl)
+              /*    */ .failureUrl(this.failureUrl);
+            return {
+              providers: await configurer.getProviders()
+            };
+          })
+          /**/ .dependencies(StandardUserWorkflow, Simple, ConfirmationUrlHttpHandler)
+          /**/ .and()
+          .apply(this);
+      });
+
+      it(
+        `creates the account and validate it`,
+        async () => {
+          await runInWorker(this, async (workflow: StandardUserWorkflow, simple: Simple) => {
+            // WHEN
+            const resultOfSignup = await workflow.signup({
```

Comment:
It is better to indicate what it is instead of 'resultOfSignup'. Here it is a pendingConfirmation. It makes the code more readable